spec/lib/sign_in/idme/service_spec.rb

```diff
@@ -18,55 +18,167 @@
   let(:user_info) do
     OpenStruct.new(
       {
-        iss: 'https://api.idmelabs.com/oidc',
-        sub: '6400bbf301eb4e6e95ccea7693eced6f',
-        aud: 'dde0b5b8bfc023a093830e64ef83f148',
-        exp: 1_647_499_198,
-        iat: 1_647_481_182,
+        iss: idme_originating_url,
+        sub: user_uuid,
+        aud: idme_client_id,
+        exp: expiration_time,
+        iat: 1_652_141_421,
         credential_aal_highest: 2,
         credential_ial_highest: 'classic_loa3',
-        birth_date: '1950-10-04',
-        email: 'vets.gov.user+228@gmail.com',
-        fname: 'MARK',
-        social: '796104437',
-        lname: 'WEBB',
+        birth_date: birth_date,
+        email: email,
+        fname: first_name,
+        social: ssn,
+        lname: last_name,
         level_of_assurance: 3,
         multifactor: true,
         credential_aal: 2,
         credential_ial: 'classic_loa3',
-        uuid: '6400bbf301eb4e6e95ccea7693eced6f'
+        uuid: user_uuid
       }
     )
   end
-  let(:success_callback_url) { 'http://localhost:3001/auth/login/callback?type=logingov' }
-  let(:failure_callback_url) { 'http://localhost:3001/auth/login/callback?auth=fail&code=007' }
+  let(:expiration_time) { 1_652_159_422 }
```

Comment:
what does this number, and the next line(current_time), combination represent?

Reply:
These are the epoch integer times representing when the JWT was created and when the JWT will expire